### PR TITLE
Search News API

### PR DESCRIPTION
**Pull request by AsmahanM**
The user can now see all the news headlines relevant to the date they've submitted.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -16,3 +16,43 @@ today =yyyy+'-'+mm+'-'+dd;
...
+        a.setAttribute('href', articles.url) //The anchor tag will open to the url of the article selected.
+        a.setAttribute('target', '_blank') //Opens the article in a new tab.
+        a.textContent = articles.title
+        li.appendChild(a)
+        result.appendChild(li) 
+    })
+
+  });
+
+}
```
Comment (Glombort) [inline script.js:58]: Looks good and works nicely

**Review by Glombort — CHANGES_REQUESTED** [review]
Looks good, and works nicely, just the minor id change (not sure whether i should change that or what the normal practice for changes like that are)

File: index.html [hunk 2]
```diff
@@ -9,15 +9,20 @@
...
+        <input class="input" id="date-picker" type="date" name="Date-Picker"
             min="2010-01-01" max="2000-13-13">
         <input type="submit">
      </form>       
     
 </main>
+
+<div class="news-container">
+    <ul class="relevant-news"></ul>
+</div>
```
[the remark this answers is not in the document]
Comment (AsmahanM) [reply index.html:24]: My bad, this should be in main (forgot to move it after testing)

**Review by AsmahanM — COMMENTED** [review]
I'm happy with these changes, thanks!

**Pull request merged**